index.html

```diff
@@ -135,7 +142,7 @@ <h3>Protocol Methods and Options</h3>
             the message type, the semantic intention of the message. As a starting
             point, a superset of REST and PubSub can cover most standard
             communication patterns. Common methods are GET, PUT, POST, DELETE,
-            PATCH, PUBLISH, and SUBSCRIBE.
+            OBSERVE, PUBLISH, and SUBSCRIBE.

```

Comment:
OBSERVE is not a method. PATCH was.
If PATCH was not common enough, then best remove both.